tests/features/start.feature

```diff
@@ -7,6 +7,10 @@ Feature: Start server
         And I ping to the server
         Then I have no errors
 
+    Scenario: I can install server
+        When I install server
+        Then I have no errors
+
     Scenario: I can start server
         When I start server
-        Then I have no errors
\ No newline at end of file
+        Then I have no errors
```

Comment:
I think we should at least write empty tests with the `Config` API service.

Reply:
there is no api functionality for config service. eula and user_jvm_args is not user set settings